main.go

```diff
@@ -321,10 +323,20 @@ func buildCustomDockerImage(ctx context.Context, dockerClient *client.Client, do
 			"BONITA_BASE_IMAGE": baseImage},
 	}
 	if *verbose {
-		fmt.Println("Using base docker image:", baseImage)
 		fmt.Println("Building new image:", *tag)
 	}
 
+	if *registryUsername != "" && *registryPassword == "" {
+		fmt.Printf("Enter your password to access Bonita Artifact Repository corresponding to '%v':", *registryUsername)
```

Comment:
The registry might not be our Bonita Artifact Repository (eg a private registry with a mirror configuration to the Artifactory)

Reply:
Ok, I'll change this.

Reply:
> Enter your password to access the Docker Registry corresponding to '%v':

Is it ok for you?